docs/upgrade/upgrade_zonemaster_backend_ver_8.0.0.md

```diff
@@ -85,12 +85,14 @@ install -v -m 755 ./zm_rpcapi-bsd /usr/local/etc/rc.d/zm_rpcapi
 
 ### Linux
 ```sh
-sudo rm `perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-Backend")')`/patch_*.pl
+cd `perl -MFile::ShareDir=dist_dir -E 'say dist_dir("Zonemaster-Backend")'`
+sudo rm patch_*.pl patch_db_README.txt create_db_sqlite.pl initial-mysql.sql initial-postgres.sql
```

Comment:
All these files were created when installing a previous version using `cpanm`, right? Maybe we could remove them using `cpanm -U Zonemaster::Backend` before reinstaling?

Reply:
We could always include `--uninstall` to create a clean stage. There could be perl modules removed too.

Reply:
I made an update.